docs/BUILD.md

```diff
@@ -134,6 +134,32 @@ For help building a desired libcrypto on your platform, please consult the proje
 
 The `CMAKE_INSTALL_PREFIX` option can be provided when building AWS-LC to specify where AWS-LC will be installed. The install path for AWS-LC should be provided when building s2n-tls, via the `CMAKE_PREFIX_PATH` option. This will ensure that s2n-tls is able to find the AWS-LC library artifact to link with.
 
+### AWS-LC FIPS
+
+s2n-tls supports FIPS mode when built with a FIPS validated version of aws-lc. See the [AWS-LC FIPS documentation](https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/FIPS.md) for how to build a FIPS validated version of AWS-LC.
+
+### Openssl FIPS
+
+If you require FIPS, you should consider using AWS-LC. If AWS-LC would conflict with Openssl in your environment, you can use the `S2N_INTERN_LIBCRYPTO` CMake option to "intern" AWS-LC and keep it isolated to s2n-tls.
```

Comment:
This is a nit, both of these sentences are passive when they could be active. Also you might want to explain why we recommend AWS-LC.

Reply:
How would you suggest making them active?

Reply:
You should consider using AWS-LC if you require FIPS. You can use the `S2N_INTERN_LIBCRYPTO` CMake option to "intern" AWS-LC and keep it isolated to s2n-tls, if AWS-LC would conflict with Openssl in your environment.